app/utilities/schema.py

```diff
@@ -98,14 +98,15 @@ def get_allowed_languages(schema_name, launch_language):
     return [DEFAULT_LANGUAGE_CODE]
 
 
-def load_schema_from_metadata(metadata):
+def load_schema_from_metadata(*, metadata, language_code=None):
+    metadata = metadata or {}
     if schema_url := metadata.get("schema_url"):
         # :TODO: Remove before production uses schema_url
         # This is temporary and is only for development/integration purposes.
         # This should not be used in production.
 
         start = time.time()
-        schema = load_schema_from_url(schema_url, metadata.get("language_code"))
+        schema = load_schema_from_url(schema_url, language_code)
```

Comment:
The logic for `language_code` should be handled in the method. Language code is optional, so it should fetch from metadata if it is not passed in.

Reply:
Updated the method to get from metadata if the language code is not passed 👍